### ISSUE TITLE
CSS CamelCase color names

### ISSUE THREAD

**Issue by maxborowsky** [Monnoroch/ColorHighlighter#340]
Unfortunately, the plugin doesn't highlight a name of color if it is written, for example, as **DarkOliveGreen**, **MidnightBlue** etc. In reference books this format of names is often used (like [http://www.w3schools.com/colors/colors_names.asp](url)). IMHO support of this format would be useful to beginners.

![image](https://cloud.githubusercontent.com/assets/20723033/20392117/79cf84ce-ace7-11e6-8e60-450866f0f85d.png)

![image](https://cloud.githubusercontent.com/assets/20723033/20392212/e4aaa206-ace7-11e6-8b7e-e6df66bd9d36.png)

**Comment by lichtamberg** [comment]
Yeah this would be really great! I can create a pull request if this change is desired from the the maintainers?

**Comment by maxborowsky** [comment]
*Yes of course, only I did not understand why you need my permission :-) *

чт, 15 мар. 2018 г. в 16:27, lichtamberg <notifications@github.com>:
